demux/basemask.py

```diff
@@ -1,62 +1,94 @@
 # -*- coding: utf-8 -*-
-
-import sys
 import click
 import logging
 
-from pathlib import Path
+from path import Path
 import xml.etree.cElementTree as et
 
-from .utils import Samplesheet, HiSeqXSamplesheet, NIPTSamplesheet, HiSeq2500Samplesheet, MiseqSamplesheet
+from .utils import (
+    Samplesheet,
```

Comment:
I find it very useful to order items alphabetically, whenever the order does not matter. It helps detecting duplicates and allows for scanning instead of reading.

Also why is this imported from `.utils` and not something more properly named like `.samplesheet`

Reply:
I'll check if I can rename the module without messing with other things too much.